### DISCUSSION TITLE
Continued Development

### DISCUSSION THREAD

**Discussion by Sev7nrayne** [a1ex4/ownfoil#79] [General]
I've noticed that development has slowed down.
Is there any news on reengagement with this project?

**Comment by Sev7nrayne** [comment]
bugs:
![image](https://github.com/a1ex4/ownfoil/assets/134185750/5c86dfb3-01c5-46cb-a4f5-a5009bedeb33)

![image](https://github.com/a1ex4/ownfoil/assets/134185750/46e13e78-98fe-4231-80b8-6af55513aa21)

format for unrecognized games distorts the layout
![image](https://github.com/a1ex4/ownfoil/assets/134185750/51d927ae-16fd-4582-bcbd-a19cde82fec6)

**Comment by irishj** [comment]
Last update on v2 / develop was six months ago, looks like the project may be dead :(

**Comment by a1ex4** [comment]
It's just that I had no time to pick up my Switch or continue developpment on the shop recently.

The v2 branch is at a pretty good state, I just want to fix the main remaining issues (game not recognized, generally how the titledb is updated is a mess) before calling it the new version and merging into develop. I'll get around to it eventually.

**Comment by Sev7nrayne** [comment]
Hey @a1ex4 is the :v2 tag defunct?
Unable to pull new image

**Reply by a1ex4** [reply]
Hi, yes the v2 tag is no more, I retired the old code into the `v1` branch, and merged the `v2` branch to `master` so it is now the default. So you should use the `latest` tag for the stable version, and the `develop` tag if you want the code that's in the `develop` branch. Hopefully you will find some additional fixes in the new version 🙂

**Reply by Sev7nrayne** [reply]
Getting this error 

Starting ownfoil
Traceback (most recemt call last):
File "/app/app-py™, line 460, in ‹module> reading conf file 
init()
File "/app/app-py", line 26, in init
reload_config()
File "/app/app-py", line 387, in reload_conf
app_settings = load_settings()
File "/app/settings.py", line 29, in load settings
settings["titles"]['valid keys'] = valid_keys
KeyError: 'titles'

**Reply by a1ex4** [reply]
Yeah you should remove your `settings.yaml` file, there have been some changes in the structure since the v2 merge, you can just backup the file and Ownfoil will recreate a new default file with the correct structure.

**Reply by Sev7nrayne** [reply]
Looks good thank you for all your hard work and dedication over the months/year since I've been apart of this project.

**Reply by a1ex4** [reply]
Thanks, glad to see it can be useful to others.

**Reply by slavix66** [reply]
Thank you for making your work available to others.
